packages/contracts/test/Airdrop.test.ts

```diff
@@ -14,14 +14,6 @@ import {
     genInputForContract,
 } from './utils'
 import { deployUnirep } from '../src'
-import {
-    ATTESTTING_FEE,
-    EPOCH_LENGTH,
-    MAX_ATTESTERS,
-    MAX_REPUTATION_BUDGET,
-    MAX_USERS,
-    NUM_EPOCH_KEY_NONCE_PER_EPOCH,
-} from '@unirep/config'
 
 describe('Airdrop', function () {
     this.timeout(100000)
```

Comment:
Set `this.timeout(0)` if you want disable timeout.

Reply:
I think it still needs a timeout here